### DISCUSSION TITLE
Add new group

### DISCUSSION THREAD

**Discussion by yanddd** [codeigniter4/shield#1201] [Q&A] [answered]
So I want to add a "create group" feature on the admin page, but after reading through all the documentation, I couldn't find a way to add a new group. Both groups and permissions can only be managed through the authGroups.php file. Do you have any suggestions for this issue?

**Comment by warcooft** [comment, marked as answer]
You can use the settings package to override groups from the AuthGroups config., and then save new groups using the settings. 

See https://settings.codeigniter.com

**Reply by yanddd** [reply]
Can you provide me with an example of the program?

**Reply by warcooft** [reply]
See https://github.com/lonnieezell/Bonfire2/blob/b892532257ee49604075596b32881767634a978e/src/Groups/Controllers/GroupSettingsController.php#L71-L109

**Reply by yanddd** [reply]
thanks
